casper/src/test/scala/coop/rchain/casper/engine/Setup.scala

```diff
@@ -1,152 +1,8 @@
-
-    val (validatorSk, validatorPk) = context.validatorKeyPairs.head
-    val bonds                      = genesisParams.proofOfStake.validators.flatMap(Validator.unapply).toMap
-    val shardId                    = genesisParams.shardId
-    val finalizationRate           = 1
-
-    val genesis: BlockMessage = context.genesisBlock
-
-    val validatorId = ValidatorIdentity(validatorPk, validatorSk, "secp256k1")
-
-    val local: PeerNode = peerNode("src", 40400)
-
-    implicit val nodeDiscovery = new NodeDiscoveryStub[Task]
-    implicit val connectionsCell: ConnectionsCell[Task] =
-      Cell.unsafe[Task, Connections](List(local))
-    implicit val transportLayer = new TransportLayerStub[Task]
-    implicit val rpConf         = createRPConfAsk[Task](local)
-    implicit val time           = Time.fromTimer[Task]
-    implicit val currentRequests: engine.BlockRetriever.RequestedBlocks[Task] =
-      Ref.unsafe[Task, Map[BlockHash, RequestState]](Map.empty[BlockHash, RequestState])
-    implicit val commUtil = CommUtil.of[Task]
-    implicit val lab =
-      LastApprovedBlock.of[Task].unsafeRunSync(monix.execution.Scheduler.Implicits.global)
-    val kvm = InMemoryStoreManager[Task]()
-    implicit val blockStore = {
-      coop.rchain.blockstorage
-        .BlockStore[Task](kvm)
-        .unsafeRunSync(monix.execution.Scheduler.Implicits.global)
-    }
-    implicit val approvedStore = {
-      coop.rchain.blockstorage.approvedStore
-        .create[Task](kvm)
-        .unsafeRunSync(monix.execution.Scheduler.Implicits.global)
-    }
-    implicit val blockDagStorage = BlockDagKeyValueStorage
-      .create(kvm)
-      .unsafeRunSync(monix.execution.Scheduler.Implicits.global)
-    implicit val synchronyConstraintChecker     = SynchronyConstraintChecker[Task]
-    implicit val lastFinalizedConstraintChecker = LastFinalizedHeightConstraintChecker[Task]
-    implicit val blockRetriever                 = BlockRetriever.of[Task]
-
-    implicit val casperBuffer = CasperBufferKeyValueStorage
-      .create[Task](spaceKVManager)
-      .unsafeRunSync(monix.execution.Scheduler.Implicits.global)
-
-    implicit val blockProcessingQueue = Queue
-      .unbounded[Task, BlockMessage]
-      .unsafeRunSync(monix.execution.Scheduler.Implicits.global)
-
-    implicit val blockProcessingState = Ref
-      .of[Task, Set[BlockHash]](Set.empty)
-      .unsafeRunSync(monix.execution.Scheduler.Implicits.global)
-
-    implicit val casperShardConf = CasperShardConf(
-      -1,
-      shardId,
-      finalizationRate,
-      Int.MaxValue,
-      Int.MaxValue,
-      0,
-      Long.MaxValue,
-      50,
-      1,
-      1,
-      genesisParams.proofOfStake.minimumBond,
-      genesisParams.proofOfStake.maximumBond,
-      genesisParams.proofOfStake.epochLength,
-      genesisParams.proofOfStake.quarantineLength,
-      1
-    )
-  }
   private def endpoint(port: Int): Endpoint = Endpoint("host", port, port)
 
   def peerNode(name: String, port: Int): PeerNode =
```

Comment:
You can now remove this whole file. What's left it's only used on BlockRetrieverSpec.